addon/components/primitives/frost-list-item-primary-state.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember'
+import layout from '../../templates/components/frost-list-item-primary-state'
+import PropTypeMixin, {PropTypes} from 'ember-prop-types'
+
+export default Ember.Component.extend(PropTypeMixin, {
+  classNames: ['frost-list-item-primary-state'],
+  classNameBindings: ['state'],
+
+  propTypes: {
+    text: PropTypes.string,
+    state: PropTypes.string
+  },
+
+  getDefaultProps () {
+    return {
+      text: '',
+      state: 'neutral'
+    }
+  },
+
+  layout
```

Comment:
We have some new conventions around comment headers that I think we should put in place here.  This is an example https://github.com/ciena-frost/ember-frost-list/blob/master/addon/components/frost-pagination.js